### ISSUE TITLE
Eureka service not running in windows 

### ISSUE THREAD

**Issue by Muzammil1234** [dice-group/sask#76]
I tried to run the eureka service first as instructed in readme, its showing the "could not execute java". I have jdk 10 installed and i even tried in running through IDE after importing the project and through git bash but its not running. I am also giving the "mvn clean spring-boot:run -P local,windows".

Here is the link to error log
https://drive.google.com/open?id=1zR7MKzP33VULPrUtF2j5gVZpYelLBXL2
Here is the link to screenshot
https://drive.google.com/open?id=1O_MjpeA2I4r1nR_d3pt-iNuVaH1avV-n
https://drive.google.com/open?id=1O_MjpeA2I4r1nR_d3pt-iNuVaH1avV-n

**Comment by KHaack** [comment]
We currently not supporting java10 (see https://github.com/dice-group/sask/issues/57) 

You can try to change the pom config to java 10 and hope that everything works :p

In the java10 branch travis was able to build and test the application.

**Comment by Muzammil1234** [comment]
It's Guru here:

Java version is not an issue, apparently our code works with Java10 too, atleast for eureka-server when we tested.

The actual issue was the port 8761 was hogged by another process, but with the logback-spring.xml enabled these errors were not apparent. Once I took away the logback-spring.xml, ran the program, I could figure out the issue.

 I have faced similar issues for the past few days wherein the having logging enabled via logback-spring.xml is masking the actual error messages.
